packages/forms/stories/story-types.ts

```diff
@@ -22,6 +22,15 @@ export interface ICheckboxStoryProps {
   validation?: VALIDATION;
 }
 
+export interface IFieldsetStoryProps {
+  isRegular: boolean;
```

Comment:
Per https://github.com/zendeskgarden/react-components/pull/1076#discussion_r621486362, remove `isRegular`.

Per previous conversations, the `Fieldset` context should be leveraged so that the `Label` for any child element changes the default of `isRegular={true}`. In other words radios, checkboxes, and toggles below a `Legend` must display with non-bold text, by default. This means a user could override those elements with `isRegular={false}` to obtain bold text, but in general this is undesired from a design POV.

Reply:
I pushed up a [new commit](https://github.com/zendeskgarden/react-components/pull/1076/commits/05d3573d02cdeca033d95a7d128d4c01792a1485) which allows labels within a `Fieldset` to render regular label styling by default. It can be overwritten with `isRegular={false}`.